src/monitoring_service/schema.sql

```diff
@@ -38,14 +38,14 @@ CREATE TABLE monitor_request (
     channel_identifier      HEX_INT     NOT NULL,
     token_network_address   CHAR(42)    NOT NULL,
 
-    balance_hash            CHAR(34)    NOT NULL,
+    balance_hash            CHAR(66)    NOT NULL,
     nonce                   HEX_INT     NOT NULL,
-    additional_hash         CHAR(32)    NOT NULL,
-    closing_signature       CHAR(34)    NOT NULL,
+    additional_hash         CHAR(66)    NOT NULL,
+    closing_signature       CHAR(132)   NOT NULL,
 
-    non_closing_signature   CHAR(160)   NOT NULL,
+    non_closing_signature   CHAR(132)   NOT NULL,
     reward_amount           HEX_INT     NOT NULL,
-    reward_proof_signature  CHAR(42)    NOT NULL,
+    reward_proof_signature  CHAR(132)   NOT NULL,
```

Comment:
Those were from the old code base. They looked suspicious, but I never got around to check them. Thanks for fixing this!

Reply:
Strange that is worked anyways, shouldn't the values be truncated?

Reply:
Not on SQLite. It takes types more as a suggestion rather than anything strict, which is really odd for and SQL database. See https://www.sqlite.org/datatype3.html for details.